package.json

```diff
@@ -22,6 +22,7 @@
     "notify:docs": "node ./scripts/notify-docs/published-ui-kit.mjs"
   },
   "dependencies": {
+    "coveo.analytics": "^2.27.12",
```

Comment:
This should not be here.
`npm i coveo.analytics@xyz` should be executed from ui-kit/packages/headless and the dependency  should be updated in `ui-kit/packages/quantic/package.json`

Reply:
Yeah I forgot to cleanup this, will remove